packages/core/mesh/network-manager/src/testing/test-builder.ts

```diff
@@ -107,18 +117,22 @@ export class TestPeer {
           exposed: {
             BridgeService: schema.getService('dxos.mesh.bridge.BridgeService'),
           },
-          handlers: { BridgeService: new WebRTCTransportService() },
+          handlers: { BridgeService: new SimplePeerTransportService() },
           noHandshake: true,
           encodingOptions: {
             preserveAny: true,
           },
         });
-
-        transportFactory = new WebRTCTransportProxyFactory().setBridgeService(this._proxy.rpc.BridgeService);
+        log.info('using SimplePeerTransportProxy');
+        transportFactory = new SimplePeerTransportProxyFactory().setBridgeService(this._proxy.rpc.BridgeService);
       } else {
-        transportFactory = {
-          createTransport: (params) => new WebRTCTransport(params),
-        };
+        if (transport === 'libdatachannel') {
+          log.info('using LibDataChannelTransport');
+          transportFactory = createLibDataChannelTransportFactory();
+        } else {
```

Comment:
Instead of catch-all else statements lets error on unexpected values. A switch-case syntax works well here